Src/library/base/elks/kernel/type.e

```diff
@@ -1,12 +1,13 @@
 ﻿note
 	description: "Representation of an Eiffel type."
+	remark: "At any given time, there is no more than one instance of TYPE representing a given Eiffel type."
```

Comment:
If there is a problem with expanded generic types, how does it work currently with class TYPED_POINTER?
The absence of the singleton is probably forward-compatible, but it is not backward compatible.